scripts/guides_master.py

```diff
@@ -27,6 +27,10 @@
             "path": "semantic_segmentation_deeplab_v3_plus",
             "title": "Semantic Segmentation with KerasCV",
         },
+        {
+            "path": "segment_anything_in_keras_cv",
+            "title": "Segment Anything in KerasCV",
```

Comment:
You can also add the keras 3 tag.

Reply:
I think all the guides under `guides/` are supposed to work with Keras 3. We didn't add this tag when porting the guides to Keras 3. Also, I don't see the tag anywhere in the file. Do we need to add this tag to other guides too?

Reply:
Ok, let's skip then!